source/includes/stackpath/_workloads.md

```diff
@@ -117,6 +117,103 @@ Attributes | &nbsp;
 `isRemoteManagementEnabled`<br/>*boolean* | Specify if remote management is enabled on workload instance or not.
 `image`<br/>*string* | The workload's instance operating system image.
 
+<!-------------------- EDIT A WORKLOAD -------------------->
+
+### EDIT a workload
+
+```shell
+curl -X PUT \
+  -H "MC-Api-Key: your_api_key" \
+  -d "request_body" \
+  "https://cloudmc_endpoint/v1/services/stackpath/test-area/workloads/1b932678-1038-4ab4-9fa4-c4c06e696e20"
+```
+> Request body example for a VM:
+```json
+{
+  "name": "my-vm-workload",
+  "stackId": "2f661cf6-8d08-42d0-918c-c20362fc9940",
+  "slug": "my-vm-workload",
+  "version": "8",
+  "type": "VM",
+  "isRemoteManagementEnabled": false,
+  "image": "stackpath-edge/centos-7-cpanel:v201905241955",
+  "cpu": "1",
+  "memory": "2Gi",
+  "specs": "SP-1",
+  "firstBootSshKey": "ssh-rsa...",
+  "deploymentName": "chicago-1",
+  "deploymentPops": "ORD",
+  "enableAutoScaling": true,
+  "cpuUtilization": 50,
+  "minInstancesPerPop": 2,
+  "maxInstancesPerPop": 3,
+  "id": "4b23edf3-9847-4400-b779-f94203227324",
+  "status": "ACTIVE"
+}
+```
+
+> Request body example for a Container:
+```json
+{
+  "name": "my-container-workload",
+  "stackId": "2f661cf6-8d08-42d0-918c-c20362fc9940",
+  "slug": "my-container-workload",
+  "version": "1",
+  "type": "Container",
+  "isRemoteManagementEnabled": false,
+  "image": "redis:alpine",
+  "addImagePullCredentialsOption": true,
+  "containerUsername": "username",
+  "containerServer": "container.server.io",
+  "containerEmail": "myemail@email.com",
+  "containerPassword": "my-password",
+  "cpu": "2",
+  "memory": "4Gi",
+  "specs": "SP-2",
+  "deploymentName": "toronto-1",
+  "deploymentPops": "YYZ",
+  "enableAutoScaling": false,
+  "deploymentInstancePerPops": 2,
+  "id": "e5f95455-1b50-4da1-86e1-6f9293f818a9",
+  "status": "ACTIVE"
+}
+```
+
+<code>PUT /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/networkpolicyrules/:id</code>
+
+Edit a new network policy rule.
+
+Required | &nbsp;
+------- | -----------
+`name`<br/>*string* | The name of the workload.
+`stackId`<br/>*UUID* | The ID of the stack that a workload belongs to.
+`slug`<br/>*string* | A workload's programmatic name. Workload slugs are used to build its instances names.
+`version`<br/>*string* | A version number for the workload. Versions start at 1 when they are created and increment by 1 every time they are updated.
+`type`<br/>*string* | Specify whether a workload is a VM-based workload or container-based.
```

Comment:
Do we have possible values for this, for example `vm` or `container`?

Reply:
e4c7e8a, 6572183